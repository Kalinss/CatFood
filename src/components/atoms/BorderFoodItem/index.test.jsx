import React from "react";
import { BorderFoodItem } from "./index";
import renderer from "react-test-renderer";
import { configure } from "enzyme";
import Adapter from "enzyme-adapter-react-16";

configure({ adapter: new Adapter() });

describe("BorderFoodItem", () => {
  it("should be rendered", () => {
    const tree = renderer.create(<BorderFoodItem />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
