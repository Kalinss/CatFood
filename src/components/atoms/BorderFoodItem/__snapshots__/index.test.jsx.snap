// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BorderFoodItem should be rendered 1`] = `
<svg
  className="svg"
  id="Слой_2"
  viewBox="0 0 320 480"
  x="0px"
  xmlSpace="preserve"
  y="0px"
>
  <path
    className="path"
    d="M 305 478 H 12.3 C 6.6 478 2 473.4 2 467.7 V 45 c 0 -3 5 -6 10 -11 c 8.5 -8.5 17.4 -17.5 25 -25 c 5 -5 5.1 -7 8 -7 h 254 c 14.5 -0.5 20.5 -0.5 19 17 v 439 C 317.5 472 318 478 305 478 Z"
  />
</svg>
`;
